tests/test_pycode/test_pycode_ast.py

```diff
@@ -51,6 +51,9 @@
      "lambda x=0, /, y=1, *args, z, **kwargs: ..."),  # posonlyargs
     ("0x1234", "0x1234"),                       # Constant
     ("1_000_000", "1_000_000"),                 # Constant
+    ("Tuple[:,:]", "Tuple[:, :]"),              # Index, Subscript, 2x Slice
```

Comment:
I'm a bit nitpicky, but do you think it's prefereable to have a space after `,` ? Also, could you perhaps add some tests using numpy extended syntax where you have commas + slices?

Reply:
No problem. Maybe the example isn't the best one, but I still think that space is okay (the space would be added automatically when reformatting a document e.g. using black). *The space is not added by the `visit_Slice`*